src/components/FormRenderer/components/FormTemplate/index.tsx

```diff
@@ -36,6 +33,7 @@ const FormTemplate: FunctionComponent<RenderProps> = ({ formFields, schema, isSu
         header,
         description,
     } = schema;
+    const isSubmitting = useContext(SubmittingContext);
```

Comment:
Let us export the `useContext(...)` in the index.tsx and import the `useFormRendererContext` from the consumers